yarn.lock

```diff
@@ -7040,30 +7054,10 @@ init-package-json@^3.0.2:
     validate-npm-package-license "^3.0.4"
     validate-npm-package-name "^4.0.0"
 
-inquirer@^8.0.0:
```

Comment:
I got a little fiddly here to de-dupe these transitive dependencies, so we have 1 less thing downloaded/installed.